docs/tools/CLI/build_profiles.md

```diff
@@ -1,42 +1,42 @@
 
-### Develop profile
+### Develop
 
 - Small and fast code.
 - Full error information. For example, asserts have file name and line number.
 - Hard to follow code flow when using a debugger.
 - Chip goes to sleep when idle:
-    * Debugger is likely to drop connection.
-    * Breaks the local file system on the [Arm Mbed interface](../introduction/index.html) on some boards.
+   - Debugger is likely to drop connection.
+   - Breaks the local file system on the [Arm Mbed interface](../introduction/index.html) on some boards.
 
-### Debug profile
+### Debug
 
-- Largest and slowest profile.
+- Largest and slowest performance.
 - Full error information. For example, asserts have file name and line number.
 - Easy to step through code with a debugger.
 - Disabled sleep mode.
 
-### Release profile
+### Release
 
-- Smallest profile and still fast.
+- Smallest code size and still fast.
 - Minimal error information.
 - Chip goes to sleep when going idle:
-    - Debugger is likely to drop connection.
-    - Breaks the local file system on the [Mbed interface](../introduction/index.html) on some boards.
+   - Debugger is likely to drop connection.
+   - Breaks the local file system on the [Mbed interface](../introduction/index.html) on some boards.
 
-### User defined profiles
+### User-defined build profile
 
-A toolchain or build system profile is a set of flags that is guaranteed to be passed to the underlying compiler suite.
+As mentioned above, the __build profile__ defines the set of flags that is guaranteed to be passed to the underlying compiler suite.
 
 These flags are stored in a JSON file that may be merged with other JSON files of the same structure.
 
-#### JSON toolchain profile format
+#### JSON build profile format
 
-The JSON object that represents a toolchain profile is a dictionary mapping from toolchains, such as `GCC_ARM`, to their flags, such as `-O3`.
+The JSON object that represents toolchain configurations for mapping each supported toolchains, such as `GCC_ARM`, to their flags, like `-O3`.
 
-The structure is as follows: each toolchain supported by a toolchain profile has a dictionary in the root dictionary. This dictionary contains a mapping from a flag type to a list of flags that should be passed to the corresponding part of the compiler suite.
+Each toolchain to be supported has an object in the root object. This object contains a mapping from a flag type to a list of flags that should be passed to the corresponding part of the compiler suite.
```

Comment:
Does "This object" in the second sentence refer to the root object or the object in the root object? Please clarify.